test/benchmarks/mdc/checkbox/app.module.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {MatCheckboxModule} from '@angular/material-experimental/mdc-checkbox';
+
+/** component: mdc-checkbox */
+
+@Component({
+  selector: 'app-root',
+  template: `
+    <button id="show" (click)="show()">Show</button>
+    <button id="hide" (click)="hide()">Hide</button>
+    <button id="indeterminate" (click)="indeterminate()">Indeterminate</button>
+
+    <mat-checkbox *ngIf="isVisible"
+      [checked]="isChecked"
+      [indeterminate]="isIndeterminate"
+      (change)="toggleIsChecked()">
```

Comment:
Is this `(change)` output needed here? I'd expect the checkbox to toggle without it too.

Reply:
It is needed to have the checkbox maintain state after being hidden.

Reply:
I see. If we keep it that way we should probably have a comment for it. What do you think?

Also did you explore just leaving it as is. Would it be an issue if hiding/showing resets the state?

Reply:
Currently I use this to set up the test for render speed of a checked checkbox

Reply:
Okay, that's fair. I'm fine keeping this as is (just a comment would be helpful I assume; but not required).